packages/storybook/stories/va-combo-box-uswds.stories.jsx

```diff
@@ -115,3 +136,8 @@ WithMessageAriaDescribedBy.args = {
   ...defaultArgs,
   messageAriaDescribedby: 'This is example aria message',
 };
+
+export const OptGroups = Template.bind({});
```

Comment:
Please expand the name of this template variable to "OptionGroups", as Storybook will automatically expand this to "Option Groups". "Opt Groups" is not "obvious" enough, in my opinion.